### DISCUSSION TITLE
How can ensure that End User adoption data from CNCF's nascent End User TAB is able to be leveraged by TAG's

### DISCUSSION THREAD

**Discussion by halcyondude** [cncf/toc#1220] [Ideas]
_spawned from:  https://github.com/cncf/toc/discussions/1210, [comment](https://github.com/cncf/toc/discussions/1210#discussioncomment-7804623)_

---

End User Adoption - related to the End User TAB. to me this means we need to ensure the TAGs have clear bi-directional communication with the End User TAB and its groups, as appropriate.

**Comment by TheFoxAtWork** [comment]
@rochaporto @kgamanji @dzolotusky Thoughts? perspectives?  Is this something that the End User TAB can take up and provide direction and structure back to the TAGs, and subsequently the TOC for our own awareness?

**Reply by rochaporto** [reply]
This is in the [TAB charter](https://github.com/cncf/foundation/blob/main/charter.md#8-end-user-technical-advisory-board-end-user-tab):
iii. Improve visibility into end user adoption of CNCF projects.
iv. Provide feedback on CNCF projects' usability, reliability, and performance.

Absolutely there needs to be a link with the TOC (already there with the two members in the TAB) but also ensuring whatever gets published is directly available and communicated to the TAGs. Once we have the TAB kicked off we could setup a call together to see how to better engage with each other?

There's another related item in the charter, on identifying gaps:
ii. Provide advice, feedback, and recommendations to the TOC on areas of mutual concern, including the state and technical direction of the cloud native ecosystem, and identification of technical challenges and gaps.

The TAB should play a big role here, giving end user feedback to the TAGs so they can help fill in those gaps and foster any new collaboration between projects or new developments.

**Comment by joshgav** [comment]
One simple form of collaboration we could implement quickly would be for TAB to hold open meetings that TOC/TAG members could listen in on and learn from.

**Comment by castrojo** [comment]
Do we have the end user content mentioned in the maintainer newsletter and other comms that we send to projects?
